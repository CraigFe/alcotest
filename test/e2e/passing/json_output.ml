let () =
  let open Alcotest in
  let id () = () in
  run ~json:true "suite-name"
    [ ( "test-a",
        [ test_case "First test case" `Quick id;
          test_case "Second test case" `Quick id
        ] );
      ("test-b", [ test_case "Third test case" `Quick id ])
    ]
