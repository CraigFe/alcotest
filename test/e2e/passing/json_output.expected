{"success":3,"failures":0,"time":0.000000}
Testing suite-name.
This run has ID `<uuid>`.
